src/Learners/Runners/CommandLine/Classifier/ClassifierEvaluationModule.cs

```diff
@@ -533,7 +533,7 @@ private void WriteAucMatrix(StreamWriter writer, IDictionary<string, IDictionary
                 writer.WriteLine("# Predicted probability, empirical probability");
                 foreach (var point in calibrationCurve)
                 {
-                    writer.WriteLine("{0}, {1}", point.First, point.Second);
+                    writer.WriteLine("{0}, {1}", point.Item1, point.Item2);
```

Comment:
This code should be using a custom struct with fields "PredictedProbability" and "EmpiricalProbability"